<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dokodemo Send</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newPacketButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New Packet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delPacketButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendAllButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="packetTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>361</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>21</number>
    </attribute>
    <column>
     <property name="text">
      <string>No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dest. IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Capture</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Length</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hex</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="sendSelectedButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Selected</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_as_pcp_2"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as_pcp">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as_pcp_2">
   <property name="text">
    <string>Save as pcap</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
